packages/concerto-tools/test/codegen/fromJsonSchema/cto/jsonSchemaVisitor.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const chai = require('chai');
+const deepEqualInAnyOrder = require('deep-equal-in-any-order');
+chai.should();
+chai.use(deepEqualInAnyOrder);
+const { assert } = chai;
+const fs = require('fs');
+const path = require('path');
+const Printer = require('@accordproject/concerto-cto').Printer;
+
+const JsonSchemaVisitor = require(
+    '../../../../lib/codegen/fromJsonSchema/cto/jsonSchemaVisitor'
+);
+const { JsonSchemaModel } = require(
+    '../../../../lib/codegen/fromJsonSchema/cto/jsonSchemaClasses'
+);
+
+const jsonSchemaVisitor = new JsonSchemaVisitor();
+const jsonSchemaVisitorParameters = {
+    metaModelNamespace: 'concerto.metamodel@1.0.0',
+    namespace: 'com.test@1.0.0',
+};
+
+describe('JsonSchemaVisitor', function () {
```

Comment:
We seem to have lost a number of the tests from the old `inferModel` file. Are they all included in your single test file now?

Reply:
Addressed ✅